db/migrate/20180111182959_rename_users_verified_to_email_verified.rb

```diff
@@ -0,0 +1,5 @@
+class RenameUsersVerifiedToEmailVerified < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :users, :verified, :verified_email
```

Comment:
Woah `:verified_email` is an interesting field type!

Reply:
What do you mean?

Reply:
Oh pff, ignore. I read that as `add_column`, not `rename_column`.